client/components/widget/query/builder/picklist-template.html

```diff
@@ -0,0 +1,13 @@
+<div class="perfkit-popup perfkit-popup-autocomplete" tabindex="-1">
+  <div class="perfkit-popup-item"
+       ng-class="{'perfkit-popup-item-unmatched': (
+          ngModel.length != 0 && ngModel != null &&
+          data.slice(0, ngModel.length).toUpperCase() != ngModel.toUpperCase()),
```

Comment:
This file was refactored from an inline HTML template, and it otherwise unchanged.  Added [https://github.com/GoogleCloudPlatform/PerfKitExplorer/issues/119] to track this improvement (though we may switch to the material design autocomplete and deprecate this in its entirety).